tests/v2/codecoverage/package/run_rancher.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Am I correct assuming most of the code in the scripts of this folder comes from existing scripts in the repo?

Reply:
Yes, some of them are mine. Like the setup environment ones, but the others I pulled from the main rancher/rancher/scripts dir.

Reply:
Cool, thanks for confirming